### ISSUE TITLE
Add code block syntax highlighting to the README

### ISSUE THREAD

**Issue by Bouncey** [freeCodeCamp/guides#322]
Make contributors aware of syntax highlighting for different languages in code blocks 

` ```html`
```html
<div class='awesome' id='more-awesome'>
  <p>This is text in html</p>
</div>
```

` ```js`
```js
function logTheThings(stuff) {
  console.log(stuff);
}
```

` ```css`
```css
.awesome {
  background-color: #BADA55;
}
```
etc...

**Comment by shovanch** [comment]
@Bouncey Should it be added under https://github.com/freeCodeCamp/guides#formatting-example-code ?

**Comment by Bouncey** [comment]
Sounds good to me @shovanch 

We should remove the line that says:
> Use two spaces to indent

**Comment by shovanch** [comment]
@Bouncey OK, I'm on it

**Comment by systimotic** [comment]
Closed through #322 🙌 Thanks @shovanch!
